src/chrome/content/rules/Switch.ch.xml

```diff
@@ -68,7 +68,6 @@
 <ruleset name="switch.ch">
 	<target host="switch.ch" />	
 	<target host="www.switch.ch" />
-	<target host="rr.aai.switch.ch" />
```

Comment:
Please add this in the comments block.

Reply:
This target still exists I only removed the duplicated entry